driver/src/main/java/org/neo4j/driver/v1/types/Point.java

```diff
@@ -22,19 +22,20 @@
 import org.neo4j.driver.v1.util.Immutable;
 
 /**
- * Represents a single tree-dimensional point in a particular coordinate reference system.
+ * Represents a single point in a particular coordinate reference system.
  * <p>
- * Value that represents a 3D point can be created using {@link Values#point3D(long, double, double, double)} method.
+ * Value that represents a point can be created using {@link Values#point(int, double, double)} method.
```

Comment:
Shall we also add a reference to the `Values#point(int, double, double, double)` overload?